content/en/agent/guide/gcp-private-service-connect.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Connect to Datadog over GCP Private Service Connect
+kind: guide
+further_reading:
+    - link: '/integrations/google_cloud_platform/'
+      tag: 'Documentation'
+      text: 'Datadog-Google Cloud Platform Integration'
+    - link: '/agent/guide/private-link'
+      tag: 'Documentation'
+      text: 'Connect to Datadog over AWS PrivateLink'
+---
+
+{{% site-region region="us,us3,eu,gov,ap1" %}}
+<div class="alert alert-warning">This feature is not supported for the selected Datadog site.</div>
+{{% /site-region %}}
+
+{{% site-region region="us5" %}}
+
+[Private Service Connect][1] (PSC) allows you to connect directly to Datadog without using the public internet.
```

Comment:
It's not clear to me what entity is connecting to Datadog, but perhaps its clear to users of GCP.

Reply:
good point, going to modify this a little bit

```suggestion
[Private Service Connect][1] (PSC) allows you to send telemetry to Datadog without using the public internet.
```